ui-ngx/src/app/modules/home/components/profile/tenant/default-tenant-profile-configuration.component.html

```diff
@@ -725,8 +733,8 @@
           </tb-rate-limits>
         </div>
         <div class="flex flex-1 flex-row xs:flex-col gt-xs:gap-4">
-          <tb-rate-limits class="flex-1" formControlName="calculatedFieldDebugEventsRateLimit"
-                          [type]="rateLimitsType.CALCULATED_FIELD_DEBUG_EVENT_RATE_LIMIT">
```

Comment:
Why was this limit removed?

Reply:
My bad that I didn't mention this in the PR description. There is no such limit on the BE side. Open any environment and try to save this limit. Nothing will happen.